src/styles/blocks/global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px $padding-vertical;
+  justify-content: center;
```

Comment:
By requirements should be only 4 card in a row.
![2023-01-20_16h04_39](https://user-images.githubusercontent.com/61904995/213715794-83e69543-24e7-4cef-8ece-a7a58d6700bb.png)